fuzz_lightyear/request.py

```diff
@@ -325,3 +331,12 @@ def _merge_kwargs(*args: Any) -> Dict[str, Any]:
     output['_request_options']['headers'] = headers
 
     return output
+
+
+def _get_auth_header(func: Callable[..., Dict[str, Any]], op_id: str) -> Dict[str, Any]:
+    header_args = inspect.getfullargspec(func.__wrapped__)  # type: ignore
+    if 'operation_id' in header_args.args:
+        auth_header = func(op_id)
```

Comment:
What if we did `func(operation_id=op_id)` to be safer, if the parameter isn't the first parameter?

Reply:
This breaks `inject_user_defined_variables` since we treat kwargs differently from args - I think keeping it like this should be fine because it's not too hard for the user to change parameter ordering.